packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -79,12 +79,12 @@ const defaultProps = {
   deceasedDateLabel: 'Deceased',
   gender: '--',
   gestationalAge: null,
-  gestationalAgeLabel: null,
+  gestationalAgeLabel: 'GA',
   identifiers: {},
   personName: '--',
   photo: null,
   postMenstrualAge: null,
-  postMenstrualAgeLabel: null,
+  postMenstrualAgeLabel: 'PMA',
```

Comment:
We do have another issue logged that will create a localized version of this banner. https://github.com/cerner/terra-core/issues/363

As for the current banner, since it's not localized, we decided to default it to the english translation.